SbtProject/project/sbt-updates.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.0")
```

Comment:
I tend not to be a fan of defining sbt plugins per project unless they define behavior specific to the project (i.e. how it's built, etc). Plugins like sbt-updates I would prefer be kept in the user's personal global build.sbt.